code/src/Form.js

```diff
@@ -0,0 +1,258 @@
+import React, { useState } from 'react';
+import { Summary } from './Summary';
+
+// Two arrays which are mapped over in the 
+const yesno = ["Yes", "No"];
+const securitySystemOptions = ["Proximity detection system", "Camera monitoring", "Security door", "Landmines", "Safe room", "A camera but you made it look like it’s just a normal rock", "Key under rock", "A moat", "Sharks with frickin' laser beams attached to their heads", "Guard dog", "A sign that says “Guard dog” and there's a picture of a German Shepherd but in reality the dog is just a Shih tzu (classic)", "Doorbell camera"];
```

Comment:
Could this be written like - to get a better over view?
const securitySystemOptions = [
"Proximity detection system", 
"Camera monitoring",
"Security door", 
"Landmines",
 "Safe room",
 "A camera but you made it look like it’s just a normal rock", 
"Key under rock", 
"A moat", 
"Sharks with frickin' laser beams attached to their heads", 
"Guard dog", 
"A sign that says “Guard dog” and there's a picture of a German Shepherd but in reality the dog is just a Shih tzu (classic)", "Doorbell camera"
];

Reply:
Agree! Or, even better - it could be in its own file 🤩